crowbar_framework/lib/crowbar/deployment_queue.rb

```diff
@@ -197,25 +193,21 @@ def process_queue
 
     private
 
-    def commit_proposals(list)
-      list.map do |item|
-        logger.debug("process queue: item to do: #{item.inspect}")
-        bc = item[:barclamp]
-        inst = item[:inst]
+    def commit_proposal(bc, inst)
+      logger.debug("process queue: committing item: #{bc}:#{inst}")
 
-        service = eval("#{bc.camelize}Service.new logger")
+      service = eval("#{bc.camelize}Service.new logger")
 
-        # This will call apply_role and chef-client.
-        # Params: (inst, in_queue, validate_after_save)
-        status, message = service.proposal_commit(inst, true, false)
+      # This will call apply_role and chef-client.
+      # Params: (inst, in_queue, validate_after_save)
+      status, message = service.proposal_commit(inst, true, false)
 
-        logger.debug("process queue: item #{item.inspect}: results #{message.inspect}")
+      logger.debug("process queue: committed item #{bc}:#{inst}: results = #{message.inspect}")
 
-        # FIXME: this is perhaps no longer needed
-        $htdigest_reload = true
+      # FIXME: this is perhaps no longer needed
```

Comment:
Same, not introduced in this PR. Do you want to submit a PR for this? Or shall I do it?